CHANGELOG.md

```diff
@@ -0,0 +1,756 @@
+# mdn-bcd-collector Changelog
+
+## v3.3.1
+
+- Automatically tag new releases (#1484), 2021-09-15
+- Add RTCSessionDescription test; fix RTCDTMFToneChangeEvent (#1488), 2021-09-15
+- Bump @google-cloud/storage from 5.14.1 to 5.14.2, 2021-09-14
```

Comment:
Still a lot of bumping going on here :)

Reply:
I set it up so the bumps get moved down to the bottom of the entries, but don't have any special magic for combining these bumps together.  I could manually combine these if you'd like?

Reply:
Let's filter out all the bump commits, and if there were any filtered out, add a "Upgraded dependencies" line to the changelog.

Reply:
I opted to omit any mention of a dependency upgrade, as quite literally every release seems to feature a dependency upgrade.  All dependency bump entries have now been removed!